applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -603,6 +639,27 @@ Selecting a configuration
 Before you start testing the application, you can select one of the :ref:`matter_bridge_app_custom_configs`.
 See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information how to select a configuration.
 
+.. _matter_bridge_smart_plug_functionality:
+
+Configure the functionality of the Matter-Bridge device
+-------------------------------------------------------
+
+To enable the Matter smart plug functionality, run the following command:
+
+.. tabs::
+
+   .. group-tab:: nRF54 DKs
+
+      .. code-block:: console
+
+         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+
+   .. group-tab:: nRF70 DKs
+
+      .. code-block:: console
+
+         west build -b nrf7002dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+
 .. _matter_bridge_testing:
 
 Testing
```

Comment:
I noticed that we also have the same DK numbering issue in the testing section. The solution we applied in other samples, to make it as easy as possible, was to create tabs, similarly to UI, and duplicating the entire testing sections with updated numbering. 
So I believe this solution would apply to "testing" and "Testing with bridged device working as a client" sections.